tests/end_to_end/Kakarot/test_coinbase.py

```diff
@@ -48,6 +41,18 @@ async def test_should_revert_when_not_owner(self, coinbase, other):
             with evm_error(error):
                 await coinbase.withdraw(0xDEAD, caller_eoa=other.starknet_contract)
 
+    class TestReceive:
+        async def test_should_receive_ether(self, coinbase: Web3Contract, owner):
+            amount = 0.001
```

Comment:
Maybe convert this to a fuzzing test with hypothesis

Reply:
just checking whether it's possible to send ether basically